### PR TITLE
Add QUICKSTART.md for LINUX

### PR DESCRIPTION
**Pull request by dcmartin**
This is initial process to setup `\psi` on Ubuntu LINUX versions 18.04 and 20.04; working correctly on AMD64; still to be tested on aarch64/arm64 and armhf/armv7.

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+
+## Step 1
+Install `.Net Core` version 3.1; download from [here](https://dotnet.microsoft.com/download/dotnet-core/3.1); select the appropriate **OS**
```
Comment (chitsaw) [inline QUICKSTART.md:4]: Isn't this step pretty much covered under Step 2? If so it can be removed.

**Review by chitsaw — CHANGES_REQUESTED** [review]
Thanks for taking the time to write this up!

File: QUICKSTART.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+## Step 2
+Install the SDK and runtime (**Ubuntu 18.04** or **20.04**)
+
+```
+sudo apt update -qq -y
+sudo apt install -qq -y apt-transport-https
+wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```
[the remark this answers is not in the document]
Comment (dcmartin) [reply QUICKSTART.md:11]: Clarified that instructions are for both 18.04 and 20.04

File: QUICKSTART.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+
```
[the remark this answers is not in the document]
Comment (dcmartin) [reply QUICKSTART.md:2]: Added new top-level section for LINUX with option for future OS specifications.

**Review by chitsaw — APPROVED** [review]
Thanks for the changes!

**Pull request merged**